torch/lib/TH/generic/THTensorCopy.h

```diff
@@ -14,4 +14,8 @@ TH_API void THTensor_(copyFloat)(THTensor *tensor, struct THFloatTensor *src);
 TH_API void THTensor_(copyDouble)(THTensor *tensor, struct THDoubleTensor *src);
 TH_API void THTensor_(copyHalf)(THTensor *tensor, struct THHalfTensor *src);
 
+int THTensor_(copyTransposeValid)(THTensor *tensor, THTensor *src);
+void THTensor_(copyTranspose)(THTensor *tensor, THTensor *src);
```

Comment:
Why are these in here? They're not used in any other file, nor are (and IMO shouldn't be) part of the public API (TH_API).